uaclient/entitlements/base.py

```diff
@@ -823,6 +823,21 @@ def can_disable(
                 ),
             )
 
+        applicability_status, _ = self.applicability_status()
+        if (
+            not self.entitlement_cfg
```

Comment:
Why do we need this `cfg` check here ?

Reply:
Because `applicability_status()` returns `APPLICABLE` if `not self.entitlement_cfg` which happened on jammy when I was testing :/

I was hesitant to change that logic in `applicability_status` so instead added an extra check out here. I can add a comment explaining this though

Reply:
@lucasmoura updated to add a comment here